import.py

```diff
@@ -227,7 +227,7 @@ def askUserForShapefileGroup(stem, existingShapefileGroups):
 def detectGeometryType(sf, stem):
   try:
     shapeType = next(shape for shape in sf.shapes() if shape.shapeType != 0).shapeType
-  except stopIteration:
+  except StopIteration:
```

Comment:
Just out of curiosity: is this an old bug, or a syntax change with the version upgrades?